### DISCUSSION TITLE
Disable markdown header (#, ##,###) being always links?

### DISCUSSION THREAD

**Discussion by maxfrauscher** [nuxt/content#1495] [General]
Hey is there a way to disable the functionality that a headers is also an anchor link? 

f.e. 

`
'### Header
`

Html Output:  

`<h3 id="header"><a href="/#header">Header</a><h3>`

I don't want to have an anchor link. Is there a way to set this up?

**Comment by maxfrauscher** [comment]
```
    content: {
        markdown: {
            tags: {
                // deactivates auto anchor links for header
                h1: 'h1',
                h2: 'h2',
                h3: 'h3',
                h4: 'h4',
                h5: 'h5',
                h6: 'h6'
            }
        }
    }
```

Solved it. You just have to overwrite the h tags in the nuxt.config.ts

**Comment by nobkd** [comment]
You can override the H2 to H4 to disable the generation. Refer to the [prose docs](https://content.nuxtjs.org/guide/writing/markdown/#prose)
and the [prose components list]( https://content.nuxtjs.org/api/components/prose)

#1248

**Comment by the-trumpeter** [comment]
If you've got a `_config.yml`, try dropping these in:
```
markdown: kramdown
auto_ids: false
```

( from https://stackoverflow.com/questions/58785061/how-to-remove-anchor-links-from-headers-in-markdown )

**Comment by nobkd** [comment]
This has an actual option since a while ago. But it was moved to another package:

See https://github.com/nuxt-modules/mdc?tab=readme-ov-file#configurations
`mdc.headings.anchorLinks: false` should work

**Comment by duahau13** [comment]
It is already in the configuration. Just add some lines to the nuxt.config.ts.
```
content: {
    renderer: {
      anchorLinks: false,
    }
}
```
https://content.nuxt.com/docs/getting-started/configuration#anchorlinks

**Reply by gokaybiz** [reply]
It still adds id's to heading tags.

**Reply by twk0d** [reply]
That worked for me in nuxt 4.0.3 with nuxt content 3.6.3, thanks!

**Comment by Kapysha-Pupkin** [comment]
has anyone found a solution?

**Reply by Kapysha-Pupkin** [reply]
So that the IDs are also deleted
